keylime/config.py

```diff
@@ -257,37 +269,59 @@ def getlist(component: str, option: str, section: Optional[str] = None) -> List[
 
 
 def get(component: str, option: str, section: Optional[str] = None, fallback: str = "") -> str:
+    env_value = _get_env(component, option, section)
     if not section:
         section = component
 
+    if env_value:
+        return env_value.strip('" ')
+
     return get_config(component).get(section, option, fallback=fallback).strip('" ')
 
 
 def getint(component: str, option: str, section: Optional[str] = None, fallback: int = -1) -> int:
+    env_value = _get_env(component, option, section)
     if not section:
         section = component
 
+    if env_value:
+        return int(env_value)
+
     return get_config(component).getint(section, option, fallback=fallback)
 
 
 def getboolean(component: str, option: str, section: Optional[str] = None, fallback: bool = False) -> bool:
+    env_value = _get_env(component, option, section)
     if not section:
         section = component
 
+    if env_value:
+        if env_value.lower() not in RawConfigParser.BOOLEAN_STATES:
```

Comment:
Should we just do the `lower()` once and store the value? I know it's not a huge optimization, but I hate running things twice that we don't need to.